### ISSUE TITLE
fatpacked bin/cpm does not run on a "naked" perl installaion

### ISSUE THREAD

**Issue by ufobat** [skaji/cpm#139]
There are is a dependencies in the fatpacked cpm module that are not fulfilled on my box.

```
# ./cpm
Can't locate Digest/MD5.pm in @INC (@INC contains: FatPacked::7580123=Hash ...
```

I coudn't find that dependency mentioned somwhere. I would suggest to use a pure perl version of md5 so it can compelty be fatpacked

**Comment by ufobat** [comment]
after installaing `Digest::MD5` i see this:

```
# ./cpm
Can't locate Data/Dumper.pm in @INC (@INC contains: FatPacked::9018152=HASH(0x899b28)
```

**Comment by skaji** [comment]
```
❯ corelist -a Digest::MD5 | head

Data for 2018-10-20
Digest::MD5 was first released with perl v5.7.3
  v5.7.3     2.16
  v5.8.0     2.20
  v5.8.1     2.27
  v5.8.2     2.30
  v5.8.3     2.33
  v5.8.4     2.33
  v5.8.5     2.33
```
cpm assumes you have perl installation with the complete set of core modules

**Comment by ufobat** [comment]
Ahhhh, yeah :-(
I've been using system perl of a centos installation.
